src/components/activity-list/ActivityList.tsx

```diff
@@ -135,20 +135,36 @@ const ActivityList = lazyMount(() => {
     [nativeCurrency, theme]
   );
 
+  const listRef = useRef<LegendListRef | null>(null);
```

Comment:
this wasn't necessary but it avoids us setting the ref over and over which should save some rendering